crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -477,6 +274,199 @@ impl TimelineEventKind {
         })
     }
 
+    pub(super) fn from_content(
```

Comment:
Niiice! I have removed a previous comment where I was suggesting to do that \o/.